pkgs/build-support/trivial-builders.nix

```diff
@@ -167,4 +167,30 @@ rec {
   # Copy a list of paths to the Nix store.
   copyPathsToStore = builtins.map copyPathToStore;
 
+
+  # Combine a list of derivations using symlinks.  Paths in later derivations
+  # take precedence over earlier ones.
+  #
+  # Example: create wrapper but retain all other files (man pages etc.)
+  #
+  # {
+  #   nixpkgs.config.packageOverrides = super: {
+  #     hello = pkgs.concat "hello" [
```

Comment:
As this function is added to a rather big namespace, a more distinctive name is useful.
How about `pkgs.concatPackages`?